statsd/statsd_benchmark_test.go

```
@@ -0,0 +1,212 @@
+func setupUDSClientServer(b *testing.B, options []statsd.Option) (*statsd.Client, net.Listener) {
+	sockAddr := "/tmp/test.sock"
+	if err := os.RemoveAll(sockAddr); err != nil {
+		log.Fatal(err)
+	}
+	conn, err := net.Listen("unix", sockAddr)
+	if err != nil {
+		log.Fatal("listen error:", err)
+	}
+	go func() {
+		for {
+			c, err := conn.Accept()
```

golangci-lint (1.62.2)

SA4006: this value of `err` is never used

(staticcheck)